packages/web_drivers/README.md

```diff
@@ -1,9 +1,18 @@
 ```
```

Comment:
Why escape the markdown syntax in a markdown file?

Reply:
it looks really cool when you do that :) it looks all gray. 

anyway I removed it.